tests/test_utils.py

```diff
@@ -0,0 +1,56 @@
+"""Tests for the util methods"""
+from opaque_keys.edx.keys import UsageKey
+from student.tests.factories import UserFactory
+
+
+from rapid_response_xblock.models import RapidResponseRun, RapidResponseSubmission
+from rapid_response_xblock.utils import get_run_data_for_course, get_run_submission_data
+from tests.utils import RuntimeEnabledTestCase
+
+
+class TestUtils(RuntimeEnabledTestCase):
+    """Utils method tests"""
+
+    def test_get_run_data_for_course(self):
+        """Verify that method returns list fo dicts with required fields."""
+        run = self._create_course_run()
+
+        expected = [
+            {
+                'id': run.id,
+                'created': run.created,
+                'problem_usage_key': run.problem_usage_key
+            }
+        ]
+
+        problem_runs = get_run_data_for_course(self.course_id)
+        assert list(problem_runs) == expected
+
+    def test_get_run_submission_data(self):
+        run = self._create_course_run()
+        user = UserFactory()
+        answer = "false"
+        event_data = {
+            "event": {
+                "submission": {
+                    "123456": {
+                        "correct": answer,
+                    }
+                }
+            }
+        }
+
+        submission = RapidResponseSubmission.objects.create(run=run, user=user, event=event_data)
+        expected = [[
+            submission.created, submission.answer_text, submission.user.username, submission.user.email, answer
+        ]]
+        submissions_data = get_run_submission_data(run.id)
+
+        assert submissions_data == expected
+
+    def _create_course_run(self):
```

Comment:
Please use `setUp` like it's used in `test_events.py`. This snippet creates a `RapidResponseRun` just like you're doing here: https://github.com/mitodl/rapid-response-xblock/blob/2e35d9a44d7f4a70dbd7e5ec4ef7deefed1dabae/tests/test_events.py#L48

You can just set the run to `self` and it will be available in both test cases here instead of needing to call a helper function in both places